VoxelSniperCore/src/main/java/com/github/kevindagame/snipe/SnipeTool.java

```diff
@@ -70,20 +71,20 @@ public IBrush setCurrentBrush(IBrush brush) {
     }
 
     public SnipeAction getActionAssigned(VoxelMaterial itemInHand) {
-        return actionTools.inverse().get(itemInHand);
+        return actionTools.get(itemInHand);
     }
 
-    public VoxelMaterial getToolAssigned(SnipeAction action) {
-        return actionTools.get(action);
-    }
+//    public VoxelMaterial getToolAssigned(SnipeAction action) {
+//        return actionTools.get(action);
+//    }
```

Comment:
This can go. It's not been used for a while and uncommenting won't result in valid code. It'll need a whole new implementation if we need it anyways

Reply:
https://github.com/KevinDaGame/VoxelSniper-Reimagined/pull/169/commits/6a345e92e269b654e3c28144a38e73d909822d36